conf/evolutions/default/1.sql

```diff
@@ -0,0 +1,19 @@
+-- Product schema
+
+-- !Ups
+
+ CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
you'll need to have uuid-ossp installed in your image, does it come for free in this image? if not you might get a failure here. If you are unable to have it with the docker container, just create the UUIDs manually